cli.js

```diff
@@ -282,6 +282,7 @@ function start (cabals) {
     if (!args.offline) {
       cabals.forEach((cabal) => {
         console.log('Seeding', cabal.key)
+        trackAndPrintEvents(cabal)
```

Comment:
can we print out the legend from the PR description here?
`
    @: new peer
    x: peer left
    ^: uploaded a chunk
    .: downloaded a chunk
`